justfile

```diff
@@ -4,6 +4,18 @@ test:
   source .env
   set +o allexport
   go test -count=1 -v ./pinecone
+test-integration:
+    #!/usr/bin/env bash
+    set -o allexport
+    source .env
+    set +o allexport
+    go test -count=1 -v -run TestIntegrationClient ./pinecone
+test-unit:
+    #!/usr/bin/env bash
+    set -o allexport
+    source .env
+    set +o allexport
+    go test -v -run Unit ./pinecone
```

Comment:
This runs all tests that contain `Unit` in the title.

Reply:
(Strategy lifted from `Subtests` section of `testing` documentation here:  https://pkg.go.dev/testing#hdr-Subtests_and_Sub_benchmarks)

Reply:
(See [this comment](https://github.com/pinecone-io/go-pinecone/pull/36/files#r1671376925) for more thoughts.)